### PR TITLE
[fix] Minor edits to Setting up an account

### PR DESCRIPTION
**Pull request by stoobie**
### Description of the Changes

Minor edits to Setting up an account

### PR Preview URL

After you push a commit to this PR, a preview is built and a URL to the root of the preview appears in the comment feed.

Paste here the specific URL(s) of the content that this PR addresses.

### Check List

- [x] Changes have been done against main branch, and PR does not conflict
- [x] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1070)

### CODE REVIEW

File: components/Starknet/modules/quick_start/pages/set_up_an_account.adoc [hunk 1]
```diff
@@ -100,7 +96,7 @@ Fetch account config from an already deployed account contract
 ----
 
 
-The `fetch` command supports both Argent X and Braavos smart wallets. Make sure your wallet address is already deployed and input following command to create and save the account descriptor file:
+The `fetch` command supports both Argent X and Braavos smart wallets. Make sure your wallet address is already deployed and input the following command to create and save the account descriptor file:
```
Comment (stoobie) [inline components/Starknet/modules/quick_start/pages/set_up_an_account.adoc:99]:
```suggestion
The `fetch` command supports both Argent X and Braavos smart wallets. Make sure your wallet address is already deployed and enter the following command to create and save the account descriptor file:
```

**Review by xiaolou86 — APPROVED** [review]

**Comment by xiaolou86** [comment]
LGTM

File: components/Starknet/modules/quick_start/pages/set_up_an_account.adoc [hunk 2]
```diff
@@ -9,21 +9,17 @@ For information on creating a Starknet wallet as an end user, see link:https://w
...
-=== Ensure Starkli and Scarb are installed correctly
-
-Ensure that the following commands show the version information for Starkli and Scarb:
-
+* Starkli and Scarb are installed correctly. Ensure that the following commands show the version information for Starkli and Scarb:
++
 [source, bash]
 ----
 starkli --version
 scarb --version
```
Comment (stoobie) [inline components/Starknet/modules/quick_start/pages/set_up_an_account.adoc:17]:
```suggestion
```

File: components/Starknet/modules/quick_start/pages/set_up_an_account.adoc [hunk 3]
```diff
@@ -9,21 +9,17 @@ For information on creating a Starknet wallet as an end user, see link:https://w
...
 ----
-
++
 If either command fails, see xref:environment_setup.adoc[Setting up your environment].
```
Comment (stoobie) [inline components/Starknet/modules/quick_start/pages/set_up_an_account.adoc:20]:
```suggestion
If this command fails, see xref:environment_setup.adoc[Setting up your environment].
```

File: components/Starknet/modules/quick_start/pages/set_up_an_account.adoc [hunk 4]
```diff
@@ -9,21 +9,16 @@ For information on creating a Starknet wallet as an end user, see link:https://w
 
 == Prerequisites
 
-=== Ensure Starkli and Scarb are installed correctly
-
-Ensure that the following commands show the version information for Starkli and Scarb:
-
+* Starkli is installed correctly. Ensure that the following command shows the version information for Starkli:
```
Comment (stoobie) [inline components/Starknet/modules/quick_start/pages/set_up_an_account.adoc:12]:
```suggestion
Starkli is installed correctly. Ensure that the following command shows the version information for Starkli:
```

File: components/Starknet/modules/quick_start/pages/set_up_an_account.adoc [hunk 5]
```diff
@@ -9,21 +9,16 @@ For information on creating a Starknet wallet as an end user, see link:https://w
 
 == Prerequisites
 
-=== Ensure Starkli and Scarb are installed correctly
-
-Ensure that the following commands show the version information for Starkli and Scarb:
-
+Starkli is installed correctly. Ensure that the following command shows the version information for Starkli:
```
Comment (stoobie) [inline components/Starknet/modules/quick_start/pages/set_up_an_account.adoc:12]:
```suggestion
* Starkli is installed correctly. Ensure that the following command shows the version information for Starkli:
```

File: components/Starknet/modules/quick_start/pages/set_up_an_account.adoc [hunk 6]
```diff
@@ -9,21 +9,17 @@ For information on creating a Starknet wallet as an end user, see link:https://w
 
 == Prerequisites
 
-=== Ensure Starkli and Scarb are installed correctly
-
-Ensure that the following commands show the version information for Starkli and Scarb:
-
+* Starkli and Scarb are installed correctly. Ensure that the following commands show the version information for Starkli and Scarb:
```
[the remark this answers is not in the document]
Comment (stoobie) [reply components/Starknet/modules/quick_start/pages/set_up_an_account.adoc]: @amanusk OK. I removed Scarb. PTAL now.

**Review by amanusk — APPROVED** [review]

**Pull request merged**